config.py

```diff
@@ -0,0 +1,1409 @@
+import os
+import time
+import numpy as np
+
+import matplotlib
+matplotlib.use('Qt5Agg')
+from matplotlib import pyplot as plt
+from qtpy.QtWidgets import QWidget, QVBoxLayout, QCheckBox, QTreeWidget, QTreeWidgetItem, QAbstractItemView, QPushButton
+
+from glue.config import qt_client
+from glue.core.data_combo_helper import ComponentIDComboHelper
+from glue.core.data_factories import load_data
+
+from glue.external.echo import CallbackProperty, SelectionCallbackProperty
+from glue.external.echo.qt import (connect_checkable_button,
+								   autoconnect_callbacks_to_qt,
+								   connect)
+
+from PyQt5.QtCore import QVariant, QItemSelectionModel, QAbstractItemModel, Qt
+
+from glue.config import viewer_tool
+from glue.viewers.common.qt.tool import CheckableTool, Tool
+
+from glue.viewers.common.layer_artist import LayerArtist
+from glue.viewers.common.state import ViewerState, LayerState
+from glue.viewers.common.qt.data_viewer import DataViewer
+from glue.viewers.common.qt.toolbar import BasicToolbar
+
+from glue.viewers.image.qt import ImageViewer
+
+from glue.utils.qt import load_ui
+
+from decimal import getcontext, Decimal
+
+import sys
+from glue.core import DataCollection, Hub, HubListener, Data
+from glue.core.link_helpers import LinkSame
+from glue.core.message import DataMessage, DataCollectionMessage, SubsetMessage, SubsetUpdateMessage
+
+import pandas as pd
+from pandas import DataFrame
+
+from PyQt5.QtGui import QStandardItemModel, QStandardItem
+
+from glue.icons.qt import helpers
+
+from PyQt5 import QtCore, QtWidgets, QtGui
+
+
+
+image_filename = '/Users/Dan/Downloads/w5/w5.fits'
+catalog_filename = '/Users/Dan/Downloads/w5/w5_psc.vot'
+
+#load 2 datasets from files
+catalog = load_data(catalog_filename)
+image = load_data(image_filename)
+
+dc = DataCollection([catalog,image])
+
+# link positional information
+dc.add_link(LinkSame(catalog.id['RAJ2000'], image.id['Right Ascension']))
+dc.add_link(LinkSame(catalog.id['DEJ2000'], image.id['Declination']))
+
+#Create subset based on filament mask
+ra_state = (image.id['Right Ascension'] > 44) & (image.id['Right Ascension'] < 46)
+subset_group = dc.new_subset_group('Subset 1', ra_state)
+subset_group.style.color = '#FF0000'
+
+de_state = image.id['Declination'] > 60
+subset_group1 = dc.new_subset_group('Subset 2', de_state)
+subset_group1.style.color = '#00FF00'
+
+j_state = catalog.id['Jmag'] > 14
+subset_group2 = dc.new_subset_group('Jmag Selection', j_state)
+subset_group2.style.color = '#00FF00'
```

Comment:
When you make the code into a package don't include the code above to load the data and make the links and subsets - that should remain in a config.py file for your personal use and should not be in the repository at all.